RPIPs/RPIP-49.md

```diff
@@ -91,41 +82,46 @@ Here we describe the mechanics of Node Operator deposits and validator creation,
 ### [RPIP-60: Protocol Upgrade Guardrails](RPIP-60.md)
 
 This RPIP introduces a delay after protocol upgrades have been confirmed but prior to them coming into effect, and allows the security council to veto the upgrade during that delay. The veto is justified if the upgrade meets one of the following criteria:
-* It was subject to vote manipulation
-* It is a malicious action
-* It causes clear damage to the Rocket Pool project
+* It was subject to vote manipulation.
+* It is a malicious action.
+* It causes clear damage to the Rocket Pool project.
+
+### RPL Value Capture
+
+Some form of value capture method will be included in the tokenomics rework package. Three options are being actively debated:
+* [RPL Burn](RPIP-45.md) - Use the surplus share to buy and burn RPL.
+* [RPL Buy & LP](RPIP-50.md) - Use the surplus share to buy RPL and deposit it in a liquidity pool. 
+* Direct the surplus share to vote-eligible Node Operators, proportional to their share of vote-eligible RP.
+
+For Saturn 1, the share will be directed based on vote-eligible RPL staked in megapools. There will be a vote prior to Saturn 1 going live to determine the preferred strategy starting at Saturn 2.
+
 
 ## Deployment Plan
 
 The tokenomics rework package will likely be split between two protocol upgrades: Saturn 1 and Saturn 2.
 
 ### Saturn 1
-
-* [RPIP-43: Megapools](RPIP-43.md)
-  * Including ETH-only validators.
-* [RPIP-42: Bond curves](RPIP-42.md)
-  * Framework
-  * 4ETH minimum bond
-* [RPIP-59: Deposit Mechanics](RPIP-59.md)
-  * Standard and express queues.
-* [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
-  * Splits between rETH, node operators, and vote-eligible staked RPL staked in megapools
-  * RPL issuance rewards no longer have a minimum stake required
-* RPL Value Capture - Increased share to voting Node Operators.
```

Comment:
I think the RPL value capture bullet should remain. Can be "RPL Value Capture - Increased share to vote-eligible RPL staked in megapools" to parallel Saturn 2 clearly.